src/app/docs/single-file-attachment/single-file-attachment-docs.component.html

```diff
@@ -112,6 +112,17 @@
         >
         </sky-docs-design-guideline-thumbnail>
       </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="File link"
+      >
+      <p>When users attach a file, the file name becomes a link. Users can click the link to download the file. You must bind the action to an event to prevent a dead link. </p>
```

Comment:
```suggestion
      <p>When users attach a file, the file name displays as a link beside the replace file button. You must bind an action to the click event to prevent a dead link and to allow users to download the file.</p>
```
I'd tweak the first sentence to state the placement of the link as in the Image preview item before it in the docs. I'd also combine the last two sentences to avoid the scenario where if users only read the first two sentences, they won't realize that they link doesn't work out of the box.